### ISSUE TITLE
Small paper preview everything with esc while cycling through pages

### ISSUE THREAD

**Issue by 3dphoto** [s1pierro/Papier#70]
Instead of only seeing which surface cut is selected by the triangle highlight if the user could hit escape on their keyboard to see what is going on rapidly

**Comment by s1pierro** [comment]
I don't anderstand. what kind of infos/view should be activated by escape ?

**Comment by s1pierro** [comment]
Ok I mess the title lol;

**Comment by 3dphoto** [comment]
Nope not all pages only that page but with what you see when clicking the icon top right, I click on 
![image](https://user-images.githubusercontent.com/28678758/41547210-6d2b617a-72d4-11e8-83fa-00dd83e97a34.png)

Then I see
![image](https://user-images.githubusercontent.com/28678758/41547248-8e140a86-72d4-11e8-8ba5-fd9129dd3f61.png)

When I click on large paper preview I get and acts as if I were to press esc for small paper preview
![image](https://user-images.githubusercontent.com/28678758/41547302-b60f3efc-72d4-11e8-94b1-5d07c46cf110.png)

Top right I click the icon to go back to normal mode 
![image](https://user-images.githubusercontent.com/28678758/41547355-d78fa800-72d4-11e8-9412-de4bb1a80c42.png)

Don't get me wrong the small paper preview is cool, is the description that may throw some users off: **2 pattern(s**), 1 x A4 - I know what to do because I know how to QA by pressing the surface cuts i can see the 2 patterns - don't think most users will know to do this
![image](https://user-images.githubusercontent.com/28678758/41547488-2a8b0efa-72d5-11e8-975b-0ea03cd26e43.png)

Could the user see the large paper preview in the small paper preview by pressing escape on the keyboard?
![image](https://user-images.githubusercontent.com/28678758/41547659-aa699808-72d5-11e8-990c-c5342a33319e.png)

The difference is leveraging the white border in viewing the paper. when user makes selection on the surface this will go back to view 1 by 1

**Comment by s1pierro** [comment]
you're right this is usefull. I will add, it's easy. There is everything needed to in the code
And KB shortcut does not impact on screen layout :)
I had planed to provide some thing like that:
![rect10](https://user-images.githubusercontent.com/6394332/41580431-2b84a34a-739b-11e8-979d-0d235f50d6bb.png)

but both will be added, what you talked about mean very usefull  for workflow to me.

**Comment by 3dphoto** [comment]
With your idea with the A4 icons is not a bad idea you may also want to allow the user to hover over the A4 to see how many KB this is. 

![image](https://user-images.githubusercontent.com/28678758/41606203-f54848a2-7397-11e8-9bf7-34307dd61a31.png)
